### PR TITLE
net-p2p/qbittorrent: version bump 3.3.3 & remove effectless python_target_useflag

### PR DESCRIPTION
**Pull request by RobinDX**
The USEFLAG python_target_python2_7 does not really effect.
The qbittorrent just use the default python in the system.
It can be set by eselect python.

### CODE REVIEW

**Comment by monsieurp** [comment]
@hwoarang

**Comment by RobinDX** [comment]
@monsieurp
I dont know .
This line be there from a long time ago.

File: net-p2p/qbittorrent/qbittorrent-3.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils qt4-r2 flag-o-matic
```
Comment (SoapGentoo) [inline net-p2p/qbittorrent/qbittorrent-3.3.3.ebuild]: @RobinDX, qt4-r2 is deprecated and is succeeded by qmake-utils. Maybe consider porting to EAPI=6 and qmake-utils?

**Comment by RobinDX** [comment]
@SoapGentoo 
There is no qt4-r2_src_prepare in qmake-utils.eclass,use what to replace it?

**Comment by RobinDX** [comment]
@SoapGentoo 
I dont know how use EAPI=6 at all,so I think EAPI=5 is fine.

File: net-p2p/qbittorrent/qbittorrent-3.3.3.ebuild (hunk 1 above)
Comment (hwoarang) [inline net-p2p/qbittorrent/qbittorrent-3.3.3.ebuild]: It would be nice to have it EAPI6 but it's not an urgent need. EAPI5 is fine

**Comment by hwoarang** [comment]
I don't understand what you mean by "python_target_useflag" since I can't see that in the 3.2.5-r1 ebuild. Otherwise the ebuild looks ok to me

**Comment by hwoarang** [comment]
Why did you change the default Qt implementation from Qt4 to Qt5? Such changes should be explained in your commit (and not your commit title).

**Comment by hwoarang** [comment]
Please re-submit this PR brining back the python changes and also please use the commit message space to document ebuild changes instead of the commit title.

**Pull request closed without merging**